3741/3/helpers/view_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'sinatra/base'
```

Comment:
Already required remove this line

Reply:
delete string from that and other helpers